_includes/_acquire_details.htm

```diff
@@ -77,6 +77,11 @@ <h3>Centralized exchanges</h3>
                             <img width="300" src="../assets/img/Exchanges/flypme.png" alt="Flyp.Me" />
                         </a>
                     </li>
+                    <li>
+                        <a href="https://www.myqbtc.com/trade?symbol=GRC_CNYT">
+                           <img width="300" src="../assets/img/Exchanges/qbtc.png" alt="qbtc">
```

Comment:
should the alt be capitalized?

Reply:
The other exchanges `alt=` tags barring Flyp.me were not capitalized so I figured it should also probably not be. 🤷‍♂